gradle-plugin/src/main/kotlin/com/google/devtools/ksp/gradle/KspSubplugin.kt

```diff
@@ -133,7 +141,10 @@ class KspGradleSubplugin @Inject internal constructor(private val registry: Tool
             kspTask.setDestinationDir(destinationDir)
             kspTask.mapClasspath { kotlinCompileProvider.get().classpath }
```

Comment:
should we update this to also delegate in the task?

Reply:
Done. Thanks!